src/nidcpower/examples/nidcpower_lcr_source_ac_voltage.py

```diff
@@ -0,0 +1,108 @@
+        session.lcr_measurement_time = lcr_measurement_time
+        session.lcr_custom_measurement_time = lcr_custom_measurement_time
+        session.lcr_source_delay_mode = lcr_source_delay_mode
+        session.source_delay = source_delay
+
+        with session.initiate():
+            session.wait_for_event(nidcpower.Event.SOURCE_COMPLETE, 21.0)
+            measurements = session.measure_multiple_lcr()
+            for measurement in measurements:
+                print(measurement)
+
+        session.reset()
+
+
+def _main(argsv):
+    parser = argparse.ArgumentParser(
+        description='Output the specified AC voltage and DC bias voltage, then takes LCR measurements',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument('-n', '--resource-name', default='PXI1Slot2/0', help='Resource names of NI SMUs')
+    parser.add_argument('-f', '--lcr-frequency', default=10.0e3, type=float, help='LCR frequency (Hz)')
+    parser.add_argument('-i', '--lcr-impedance-range', default=100.0, type=float, help='LCR impedance range (Ohm)')
+    parser.add_argument('-c', '--cable-length', default='NI_STANDARD_2M', type=str, choices=tuple(nidcpower.CableLength.__members__.keys()), help='Cable length')
+    parser.add_argument('-v', '--lcr-voltage-rms', default=700.0e-3, type=float, help='LCR voltage RMS (V RMS)')
+    parser.add_argument('-d', '--lcr-dc-bias-source', default='OFF', type=str, choices=tuple(nidcpower.LCRDCBiasSource.__members__.keys()), help='LCR DC bias source')
+    parser.add_argument('-dv', '--lcr-dc-bias-voltage_level', default=0.0, type=float, help='LCR DC bias voltage (V)')
+    parser.add_argument('-t', '--lcr-measurement-time', default='MEDIUM', type=str, choices=tuple(nidcpower.LCRMeasurementTime.__members__.keys()), help='LCR measurement time')
+    parser.add_argument('-ct', '--lcr-custom-measurement-time', default=10.0e-3, type=float, help='LCR custom measurement time (s)')
+    parser.add_argument('-sm', '--lcr-source-delay-mode', default='AUTOMATIC', type=str, choices=tuple(nidcpower.LCRSourceDelayMode.__members__.keys()), help='LCR source delay mode')
+    parser.add_argument('-s', '--source-delay', default=16.66e-3, type=float, help='Source delay (s)')
+    parser.add_argument('-op', '--option-string', default='', type=str, help='Option string')
+    args = parser.parse_args(argsv)
+    example(
+        args.resource_name,
+        args.option_string,
+        args.lcr_frequency,
+        args.lcr_impedance_range,
+        getattr(nidcpower.CableLength, args.cable_length),
+        args.lcr_voltage_rms,
+        getattr(nidcpower.LCRDCBiasSource, args.lcr_dc_bias_source),
+        args.lcr_dc_bias_voltage_level,
+        getattr(nidcpower.LCRMeasurementTime, args.lcr_measurement_time),
+        args.lcr_custom_measurement_time,
+        getattr(nidcpower.LCRSourceDelayMode, args.lcr_source_delay_mode),
+        args.source_delay,
```

Comment:
Specify these as keyword args; it improves readability which is essential in examples.